sssom/util.py

```diff
@@ -779,6 +805,11 @@ def to_mapping_set_dataframe(doc: MappingSetDocument) -> MappingSetDataFrame:
             m = get_dict_from_mapping(mapping)
             data.append(m)
     df = pd.DataFrame(data=data)
+    # The following 3 lines are to remove columns
+    # where all values are blank.
+    df = df.replace("", np.nan)
+    df = df.dropna(axis=1, how="all")
+    df.loc[:, df.columns != CONFIDENCE].replace(np.nan, "", inplace=True)
```

Comment:
Like in the other PR, shouldnt this be applied to all `double` columns?

Reply:
It would have changed on merging with the other branch but I've changed it here too.